src/main/java/org/broadinstitute/hellbender/utils/SimpleInterval.java

```diff
@@ -40,6 +40,16 @@ public SimpleInterval(final String contig, final int start, final int end){
         this.end = end;
     }
 
+    /**
+     * Create a new immutable 1-based interval of the form [startAndEnd, startAndEnd]
+     * that is 1 bp long.
+     * @param contig the name of the contig, must not be null
+     * @param startAndEnd  1-based inclusive start/end position (that are the same since it is only 1 bp long)
+     */
+    public SimpleInterval(final String contig, final int startAndEnd) {
```

Comment:
Funny, in my branch I have the same addition. I guess it is long due.

Reply:
We can merge them just like for the SVUtils methods

Reply:
Nah, I think it is helpful in general to create the 1bp interval without the need to indicate the position twice. I think is good here.